activity/voice/items/say_ah_covid

```diff
@@ -3,10 +3,10 @@
     ],
     "@type": "reproschema:Field",
     "@id": "say_ah",
-    "skos:prefLabel": "Say Ah",
-    "schema:description": "say ah for 10 seconds",
-    "schema:schemaVersion": "0.0.1",
-    "schema:version": "0.0.1",
+    "prefLabel": "Say Ah",
```

Comment:
i think this one should be moved over to voice task.

Reply:
yes, i needed some clarification from daniel since there were two items with similar name. will move it.